README.md

```diff
@@ -61,15 +61,19 @@ export OPERATOR_IMAGE=<registry url for remote WMCO image>
 ```
 Once the above variables are set, run the following script:
 ```shell script
-hack/run-ci-e2e-test.sh -k "openshift-dev"
+hack/run-ci-e2e-test.sh -k "openshift-dev" -b ./build/_output/bin/windows-machine-config-operator
 ```
-We assume that the developer uses `openshift-dev` as the key pair in the aws cloud
+We assume that the developer uses `openshift-dev` as the key pair in the aws cloud.
 
 Additional flags that can be passed to `hack/run-ci-e2e-test.sh` are
 - `-s` to skip the deletion of Windows nodes that are created as part of test suite run
 - `-n` to represent the number of Windows nodes to be created for test run
 - `-k` to represent the AWS specific key pair that will be used during e2e run and it should map to the private key
        that we have in `KUBE_SSH_KEY_PATH`. The default value points to `openshift-dev` which we use in our CI
+- `-b` gives an alternative path to the WMCO binary. If the -b option is not given outside of CI, certain validation
```

Comment:
- Is this is a required parameter when running locally? If yes, then it should be called out.
- Ideally it would be nice for this to be an option parameter with a sensible default.
- What validations tests fail? It would help to call that out here.

Reply:
Changed this to use the normal dev workflow as default.